package.json

```diff
@@ -57,6 +57,7 @@
     "cors": "^2.8.5",
     "dotenv": "^8.0.0",
     "errorhandler": "^1.5.1",
+    "eslint": "^7.30.0",
```

Comment:
**eslint is devDependency**. So place it in devDependency only and not under dependencies .